core/datainput.gms

```diff
@@ -1186,9 +1149,10 @@ $offdelim
 ;
 p_adj_deltacapoffset("2015",regi,"tnrs")= 1;
 
-$ifthen.WindOff %cm_wind_offshore% == "1"
-p_adj_deltacapoffset(t,regi,"windoff")= p_adj_deltacapoffset(t,regi,"wind");
-$endif.WindOff
+*** windoffshore-todo
+p_adj_deltacapoffset(t,regi,"windon") $ (p_adj_deltacapoffset(t,regi,"windon") eq 0) = p_adj_deltacapoffset(t,regi,"wind");
```

Comment:
same here - I am not sure I understand this. 
why getting windon adj-parameters from wind, if the wind values are set right below - why not simply setting the windon values and removing the "wind" entry?

Reply:
same as above, didn't understand the if statement

Reply:
The meaning is:
If mrremind data does not contain a value for *windon* (because it is still using *wind*), then define the variable for *windon* by copying the value of *wind* (and discard the *wind* variable because we won't need it in remind)

Reply:
my mistake - I had overlooked that the "deltacapoffset" part actually comes from mrremind and is not simply defined below. 
sorry!